docker/docker_image_dest.go

```diff
@@ -104,6 +122,19 @@ func (c *sizeCounter) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
+func prepareMountQuery(mounts []reference.Named, digest string) string {
+	var repositories []string
+	for _, m := range mounts {
+		// Mount only takes the path, not the hostname
+		mount := reference.Path(m)
+		repositories = append(repositories, "from="+mount)
+	}
+	if len(repositories) > 0 {
+		return fmt.Sprintf("mount=%s&%s", digest, strings.Join(repositories, "&"))
```

Comment:
AFAICT, looking both at the official API spec and at the docker/distribution implementation, the `from` parameter does not support multiple values in a single request.

Reply:
Strange, GCR definitely supports multiple, but I'm not aware of any actual use-cases for this so I dropped it.